### PR TITLE
fix: scrollbars not appearing in auth window

### LINKED ISSUE #1765: GitHub oauth popup doesn't allow scrolling in Firefox
When using the GitHub authentication backend, the popup window that displays the application authorization page doesn't allow scrolling when using Firefox. It seems to work fine in Chrome and Edge.

**To Reproduce**
1. Open the CMS admin page on a site configured to use the GitHub backend
2. Click "Login with GitHub"
3. Try to scroll the authorization request page

**Expected behavior**
Window should allow scrolling so users can see the information at the bottom.

**Screenshots**

No scrollbar in Firefox:
image

Chrome has a scrollbar:
image

**Applicable Versions:**

 - Netlify CMS version: 2.1.2
 - Git provider: GitHub
 - OS: Windows 10
 - Browser version: Firefox 63.0b8 (64-bit)

### PR DESCRIPTION
**Pull request by tech4him1**
**Summary**

Closes #1765.

This was only a problem in Firefox, since it is the only one that actually accepts `scrollbars=no`.

**Test plan**
Tested desktop browsers:
- [x] Chrome
- [x] Opera
- [x] Firefox
- [x] Edge
- [x] Safari

**A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

**Comment by verythorough** [comment]
Deploy preview for *netlify-cms-www* ready!

Built with commit ad2c4c40614075cfadc4e99eba1aaa262ba851c0

https://deploy-preview-1766--netlify-cms-www.netlify.com

**Comment by verythorough** [comment]
Deploy preview for *netlify-cms-www* ready!

Built with commit 87ea9da4c329ae1a73cd3ece32de1313aea02c58

https://deploy-preview-1766--netlify-cms-www.netlify.com

**Comment by verythorough** [comment]
Deploy preview for *cms-demo* ready!

Built with commit ad2c4c40614075cfadc4e99eba1aaa262ba851c0

https://deploy-preview-1766--cms-demo.netlify.com

**Comment by verythorough** [comment]
Deploy preview for *cms-demo* ready!

Built with commit 87ea9da4c329ae1a73cd3ece32de1313aea02c58

https://deploy-preview-1766--cms-demo.netlify.com

File: packages/netlify-cms-lib-auth/src/netlify-auth.js [hunk 1]
```diff
@@ -126,16 +126,7 @@ class Authenticator {
...
-        ('width=' +
-          conf.width +
-          ', height=' +
-          conf.height +
-          ', top=' +
-          top +
-          ', left=' +
-          left +
-          ');'),
+      `width=${conf.width}, height=${conf.height}, top=${top}, left=${left}`,
```
Comment (erquhart) [inline packages/netlify-cms-lib-auth/src/netlify-auth.js]: Any reason not to continue stripping down the auth window and just stop setting `scrollbars` to `no`? It's pretty typical for auth windows to be featureless.

Comment (tech4him1) [reply packages/netlify-cms-lib-auth/src/netlify-auth.js]: Everything is `no` by default, except things which you can't override anyway or the browser manufacturer thinks should be on. So the dropping of these other items is simply a separate cleanup, and could have been a separate commit.

Comment (erquhart) [reply packages/netlify-cms-lib-auth/src/netlify-auth.js]: Ah, I did not consider whether those were the defaults, got it 👍

**Pull request merged**